### PR TITLE
 Add Signup and Login pages

### PR DESCRIPTION
**Pull request by darshandamre**
Login page preview [https://e-com-git-auth-pages-darshandamre.vercel.app/pages/login.html](https://e-com-git-auth-pages-darshandamre.vercel.app/pages/login.html)

Signup page preview [https://e-com-git-auth-pages-darshandamre.vercel.app/pages/signup.html](https://e-com-git-auth-pages-darshandamre.vercel.app/pages/signup.html)

### CODE REVIEW

File: pages/auth.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import "../global.css";
+
+.auth__content {
+  max-width: 32rem;
+  background: var(--black);
+  border-radius: 10px;
```
Comment (arjunbharti) [inline pages/auth.css:6]: use rem

**Review by arjunbharti — APPROVED** [review]
Amazing UI and code good looks to me.

File: pages/auth.css (hunk 1 above)
Comment (darshandamre) [reply pages/auth.css:6]: i think 10px is convenient than writing 0.625rem and it also doesn't affect responsiveness to make a difference

**Review by SHIKHASINGH1506 — APPROVED** [review]
Crisp code

**Pull request merged**